cloudify_aws/ec2/resources/image.py

```diff
@@ -77,7 +77,7 @@ def create(self, params):
         return res
 
     def delete(self, params=None):
-        return
+        return None
```

Comment:
Well yeah, you are right python 3 somehow do not like this for some reason and the unit tests failed so I had to update it. maybe it did not like it because was saving the None type on a variable but its worth trying it again and if its not break the tests then will revert this back